### PR TITLE
🚀 Improve Search UI and UX

### LINKED ISSUE #250: Refactor Search Selector UI/UX in Home Page
Todo: 🚀

- [x] Design in Playground
- [x] Create the Components
- [x] Testing the UX 😂
- [x] Shipping it for Feedback

### PR DESCRIPTION
**Pull request by shrihari689**
## Proposed Changes 🎉

- Redesign Search Selector in the Home Page. 🎨
- Added Auto Completion (Enter and Tab key stuffs) 🎹
- Clean Up Code on Home Page 🧹
- Planning to reduce the use of `active_districts_v2` (or completely remove it) because,
   - since the data is not proper
   - since we have now integrated Twitter
   - since we are generating static paths for all states/districts
   - we do have a filter inside results page 🤔

Note: `topTenStatesAndDistricts` is hardcoded because its changes are less frequent and don't want to make API call that would take some time 📍

Please do review and comment 📌 I will be happy to push new changes 🎉

Resolves #250 for now 😁

### CODE REVIEW

File: lib/search.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
+import states from '@data/states.json';
+
+export const topTenStatesAndDistricts = [
```
Comment (minkesh) [inline lib/search.js:3]: can we move this to data files ?

Comment (shrihari689) [reply lib/search.js]: Done ✅

**Comment by tomahawk-pilot** [comment]
#### 📝Changed routes:


| [**/**](https://life-kqu19a0aj-coronasafe.vercel.app/) |
|:-:|
|<a href="https://life-kqu19a0aj-coronasafe.vercel.app/">Screenshot of /</a><br /><sup><a href="https://images.weserv.nl?url=https://iad.microlink.io/-iJv1Yz9sLADM4tpayO6M_gvhbF2gBRX8eo1L6-RnN78i4G2-A6UsdA-fs9egER1RRbwAM9Ix6CeDtzv7Eg0aA.png">(view full size)</a> |







Commit 4ee13b7581c1af6fb93c51eb75dd5d7a578c4444 (https://life-kqu19a0aj-coronasafe.vercel.app).

**Comment by shrihari689** [comment]
Couldn't get you @tomahawk-pilot 😢

**Comment by bodhish** [comment]
This is just Incredible 🔥

**Pull request merged**